app/includes/header.njk

```diff
@@ -19,14 +19,17 @@
   }) }}
 </span>
 
+{% if loginEnabled === 'true' %}
```

Comment:
{% if loginEnabled %} -> should also work

Reply:
it doesn't because the config is strings not booleans - this puzzled me for quite some time 😄

Reply:
I will see if I can change it in the context creator